account_invoice_overdue_reminder/wizard/overdue_reminder_wizard_view.xml

```diff
@@ -165,7 +163,7 @@
                             />
                         <field name="state" invisible="1" />
                         <field name="payment_state" invisible="1" />
-                        <field name="result_id"/>
+                        <field name="result_id" />
```

Comment:
I think your last commit of black,isort,prettier can be combined with the previous one. The first commit should remain separate.